language-server/modules/langserver-core/src/test/resources/completion/typedesc_context/source/function_typedesc17b.bal

```diff
@@ -0,0 +1,19 @@
+import ballerina/module1;
+
+public const int TEST_CONST = 12;
+
+public function main() {
+    transactional w
+    int[] testArr = [];
```

Comment:
Shall we add a test cases with comments.
example:
```
import ballerina/module1;

public const int TEST_CONST = 12;
public function main() {
    transactional w // some coment
    int[] testArr = [];
}

public type TestMap2 map<string>;
public type TestMap3 map<int>;

type TestMap1 map<int>;

type TestRecord1 record {
    int rec3Field1 = 12;
    string rec3Field2 = "TestRecord3";
};
```

Reply:
Done!